terraform-plans/modules/GitHub/workflows/main.tf

```diff
@@ -47,3 +87,12 @@ resource "github_repository_pull_request" "workflows_update_pr" {
     github_repository_file.workflows_files
   ]
 }
+
+output "changed_files" {
+  description = "Execute result: [repository, branch, changed_files]"
+  value = [
+      var.repository,
+      var.branch,
+      local.changed_files
+  ]
+}
```

Comment:
I tend to do that. Because in the real output the branch and repository information is not very clear. Also changed_files doesn't include repo and branch information.

Reply:
It's fixed.